### PR TITLE
Add Swift Workflow 0.1.0

### PR DESCRIPTION
**Pull request by timdonnelly**
Co-authored-by: David Apgar <apgar@squareup.com>

### CODE REVIEW

File: Workflow.podspec [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |s|
+  s.name         = 'Workflow'
+  s.version      = '0.1.0'
+  s.summary      = 'Reactive application architecture'
+  s.homepage     = 'https://www.github.com/square/workflow'
+  s.license      = 'Apache License, Version 2.0'
+  s.author       = 'Square'
+  s.source       = { :git => 'https://github.com/square/workflow.git' }
+
+  s.swift_versions = ['4.2']
```
Comment (dnkoutso) [inline Workflow.podspec]: if you do use this DSL you probably want to add `s.cocoapods_version >= '1.7.0.beta.1'` to ensure consumers can receive a msg.

File: Workflow.podspec [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |s|
+  s.name         = 'Workflow'
+  s.version      = '0.1.0'
+  s.summary      = 'Reactive application architecture'
+  s.homepage     = 'https://www.github.com/square/workflow'
+  s.license      = 'Apache License, Version 2.0'
+  s.author       = 'Square'
+  s.source       = { :git => 'https://github.com/square/workflow.git' }
```
Comment (dnkoutso) [inline Workflow.podspec]: you also want a `tag` here `:tag => #{s.version}`

File: Workflow.podspec [hunk 3]
```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |s|
+  s.name         = 'Workflow'
```
Comment (dnkoutso) [inline Workflow.podspec]: noone has claimed this name!

**Comment by dnkoutso** [comment]
💯 
We can also setup CI to `lint`.

File: Workflow.podspec (hunk 1 above)
Comment (dnkoutso) [reply Workflow.podspec]: Alternative you can use `s.swift_version = '4.2'` to begin with.

Comment (dnkoutso) [reply Workflow.podspec]: 1.7.x will be backwards compatible once you switch to multiple Swift versions

**Comment by timdonnelly** [comment]
> 💯
> We can also setup CI to `lint`.

@dnkoutso  What's the best strategy to set up CI for a single podspec without jumping through an app/`Podfile`?

**Comment by dnkoutso** [comment]
@timdonnelly you just run `pod lib lint <podspec>` to it.

**Comment by dnkoutso** [comment]
@timdonnelly I can help you set it up after this PR or for this PR.

File: swift/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Change Log
```
Comment (zach-klippenstein) [inline swift/CHANGELOG.md]: Can you link to this changelog from the root changelog?

File: swift/README.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Workflow
```
Comment (zach-klippenstein) [inline swift/README.md]: Are you planning to add release instructions? We've got them for [kotlin](https://github.com/square/workflow/blob/master/kotlin/RELEASING.md), and they're linked to from the root readme. If you add them, just link from there as well.

File: swift/CHANGELOG.md (hunk 4 above)
Comment (timdonnelly) [reply swift/CHANGELOG.md]: Done

File: swift/README.md (hunk 5 above)
Comment (timdonnelly) [reply swift/README.md]: Not yet, but we should add them as we figure it out.

**Pull request merged**